docs/README.Ubuntu.md

```diff
@@ -125,7 +125,7 @@ sudo apt install libcec4-dev libfmt3-dev liblircclient-dev
 
 If you're using Ubuntu 18.04, you also need to install:
 ```
-sudo apt install libcec-dev libfmt-dev liblirc-dev
+sudo apt install libcec-dev libfmt-dev liblirc-dev flatbuffers-dev
```

Comment:
`flatbuffers-dev` is not only needed for Ubuntu 18.04, it's needed everywhere. It has to be added to the common dependency list a few lines above

Reply:
You're right. Didn't test it on Ubuntu 16.04.
I'll update this pr tomorrow. Thanks.